README.md

```diff
@@ -1,707 +1,1799 @@
+  (todos, id) => todos.filter(todo => todo.id === id),
+  {
+    memoizeOptions: {
+      maxSize: 10
+    }
+  }
+)
+```
+
+But this required having to know the cache size ahead of time.
+
+2. Create unique selector instances using [`useMemo`].
+
+```tsx
+const parametricSelector = (id: number) =>
+  createSelector([(state: RootState) => state.todos], todos =>
+    todos.filter(todo => todo.id === id)
+  )
+
+const MyComponent: FC<Props> = ({ id }) => {
+  const selectTodosById = useMemo(() => parametricSelector(id), [id])
+
+  const todosById = useSelector(selectTodosById)
+
+  return (
+    <div>
+      {todosById.map(todo => (
+        <div key={todo.id}>{todo.title}</div>
+      ))}
+    </div>
+  )
+}
+```
+
+3. Using [`useCallback`].
+
+```tsx
+const parametricSelector = createSelector(
+  [(state: RootState) => state.todos, (state: RootState, id: number) => id],
+  (todos, id) => todos.filter(todo => todo.id === id)
+)
+
+const MyComponent: FC<Props> = ({ id }) => {
+  const selectTodosById = useCallback(parametricSelector, [])
+
+  const todosById = useSelector(state => selectTodosById(state, id))
+
+  return (
+    <div>
+      {todosById.map(todo => (
+        <div key={todo.id}> {todo.title}</div>
+      ))}
+    </div>
+  )
+}
+```
+
+4. Use [`re-reselect`]:
+
+```ts
+import { createCachedSelector } from 're-reselect'
+
+const parametricSelector = createCachedSelector(
+  [(state: RootState) => state.todos, (state: RootState, id: number) => id],
+  (todos, id) => todos.filter(todo => todo.id === id)
+)((state: RootState, id: number) => id)
+```
+
+Starting in 5.0.0, you can eliminate this problem using `weakMapMemoize`.
+
+###### Using [`createSelector`]
+
+```ts
+const parametricSelector = createSelector(
+  [(state: RootState) => state.todos, (state: RootState, id: number) => id],
+  (todos, id) => todos.filter(todo => todo.id === id),
+  {
+    memoize: weakMapMemoize,
+    argsMemoize: weakMapMemoize
+  }
+)
+
+parametricSelector(state, 0) // Selector runs
+parametricSelector(state, 0)
+parametricSelector(state, 1) // Selector runs
+parametricSelector(state, 0)
+```
+
+###### Using [`createSelectorCreator`]
+
+```ts
+import { createSelectorCreator, weakMapMemoize } from 'reselect'
+
+const createSelectorWeakMap = createSelectorCreator({
+  memoize: weakMapMemoize,
+  argsMemoize: weakMapMemoize
+})
+
+const parametricSelector = createSelector(
+  [(state: RootState) => state.todos, (state: RootState, id: number) => id],
+  (todos, id) => todos.filter(todo => todo.id === id)
+)
+
+parametricSelector(state, 0) // Selector runs
+parametricSelector(state, 0)
+parametricSelector(state, 1) // Selector runs
+parametricSelector(state, 0)
+```
+
+This solves the problem of having to know and set the cache size prior to creating a memoized selector.
+
+</details>
+
+<div align="right">[ <a href="#table-of-contents">↑ Back to top ↑</a> ]</div>
+
+---
+
+<a id="autotrackmemoize"></a>
+
+#### autotrackMemoize(func) - (**`Since`** 5.0.0)
```

Comment:
:hand: Let's 100% refer to this as `unstable_autotrackMemoize` everywhere in the docs until it's actually ready for release.

Reply:
Surething